docker_files/recipes/openfoam.core-2.4.0/openfoam-core.patch

```diff
@@ -1,3 +1,10 @@
+#
```

Comment:
I suggest you do not use `#` in the patch file. According to patch file specs, everything above `---` is a comment.